Dockerfile

```
@@ -0,0 +1,11 @@
+FROM ruby:3.2
+
+WORKDIR /app
+COPY . .
+
+RUN gem install bundler jekyll
+RUN bundle install
+
+EXPOSE 4000
+
+CMD ["bundle", "exec", "jekyll", "serve", "--host", "0.0.0.0"]
```

This would be good, if it works?

```suggestion
CMD ["bundle", "exec", "jekyll", "serve", "--host", "0.0.0.0", "--livereload"]
```